### DISCUSSION TITLE
Update @zeppos/device-types

### DISCUSSION THREAD

**Discussion by sk1nzz** [zepp-health/zeppos-forum#209] [Developer Feedback]
**The latest version of [@zeppos/device-types](https://www.npmjs.com/package/@zeppos/device-types) was released 9 months ago. This version is missing types for a lot of Zepp OS APIs.**

Missing constructor functions: `Watchface`, `AppSideService`, `AppSettingsPage`.
Missing watchface widget types: `TIME_POINTER`, `IMG_STATUS`, `IMG_TIME`, `IMG_DATE`, `IMG_WEEK`, `DATE_POINTER`, `IMG_CLICK`
ALL ZeppOS 2.1 new APIs are missing in the types package!

Please add these types and publish a new version of this package to make the development process easier for us.

**Comment by MyoungXUE** [comment]
@sk1nzz 
Hello, this is a very good suggestion. We are sorting out API related recently and will update @zeppos/device-types as soon as possible.

**Comment by MyoungXUE** [comment]
@sk1nzz 
Hello, we have fixed many bugs and provided many new feature announcements, please update to the latest version!

https://www.npmjs.com/package/@zeppos/device-types
